src/main/content/instance.html

```diff
@@ -219,20 +218,61 @@ <h4>{{t.instance.stacks}}</h4>
             </div>
         </div>
         <div data-notification class="bx--inline-notification">
-            <div id="modal-column-headings"
-                class="bx--inline-notification__details instance-inline-admin-list-notification">
+            <div class="modal-column-headings bx--inline-notification__details instance-inline-admin-list-notification">
                 <div class="bx--inline-notification__text-wrapper">
                     <div class="modal-content-user-info">{{t.instance.user}}</div>
                     <div class="modal-content-user-info">{{t.instance.full-name}}</div>
                     <div class="modal-content-user-info">{{t.instance.email}}</div>
+                    <div class="modal-column-heading-empty-div"></div>
+                </div>
+            </div>
+        </div>
+        <div id="addUser-input-template" class="hidden addUser-input-container">
+            <div class="bx--form-item bx--text-input-wrapper">
+                <label for="text-input-3" class="bx--label">{{t.instance.add-user}} <span
+                        class="user-add-team-name"></span>
+                    {{t.instance.team}}</label>
+                <div class="input-and-button-container">
+                    <div class="input-and-error-msg-container">
+                        <div class="bx--text-input__field-wrapper text-input-wrapper">
+                            <div class="input-error-msg-icon hidden">
+                                <svg focusable="false" preserveAspectRatio="xMidYMid meet"
+                                    xmlns="http://www.w3.org/2000/svg" class="bx--text-input__invalid-icon" width="16"
+                                    height="16" viewBox="0 0 16 16" aria-hidden="true">
+                                    <path
+                                        d="M8,1C4.2,1,1,4.2,1,8s3.2,7,7,7s7-3.1,7-7S11.9,1,8,1z M7.5,4h1v5h-1C7.5,9,7.5,4,7.5,4z M8,12.2 c-0.4,0-0.8-0.4-0.8-0.8s0.3-0.8,0.8-0.8c0.4,0,0.8,0.4,0.8,0.8S8.4,12.2,8,12.2z">
+                                    </path>
+                                    <path
+                                        d="M7.5,4h1v5h-1C7.5,9,7.5,4,7.5,4z M8,12.2c-0.4,0-0.8-0.4-0.8-0.8s0.3-0.8,0.8-0.8 c0.4,0,0.8,0.4,0.8,0.8S8.4,12.2,8,12.2z"
+                                        data-icon-path="inner-path" opacity="0"></path>
+                                </svg>
```

Comment:
move this svg to a file in img dir and include it with an img tag, looks very messy here